bigbluebutton-html5/imports/ui/components/nav-bar/settings-dropdown/container.jsx

```diff
@@ -1,59 +1,59 @@
-import React, { Component } from 'react';

```

Comment:
This file is listed as rewritten but I am not seeing the changes. What are the changes?